MoskofidiAA/lab_4/main.cpp

```diff
@@ -0,0 +1,99 @@
+#include <iostream>
+
+using namespace std;
+
+float deg (float n, int degree) {
+    int k = n;
+    n = 1;
+    for (int i = 0; i < degree; i++) {
+        n *= k;
+    }
+    return n;
+}
+
+int fact (int n) {
+    int k = 1;
+    for (int i = 2; i <= n; i++) {
+        k *= i;
+    }
+    return k;
+}
+
+double sqrt(double num)
+{
+    double m;
+    double result = num / 2;
+    do
+    {
+        m = result;
+        result = (m + (num / m)) / 2;
+
+    }
+    while ((m - result) != 0);
+    return result;
+}
+
+int simple (int n) {
+    int i(0), k(0);
+    for ( i = 2; i < n; i++) {
+        if ( n % i == 0) {
+            k++;
+        }
+    }
+    if (k == 0) {
+        cout << "Number " << n << " is simple" << endl;
```

Comment:
Function should return different values depends on whether number is prime or not.
Not just print smth to console.